FWCore/Reflection/test/BuildFile.xml

```diff
@@ -0,0 +1,8 @@
+<library file="stubs/*.cc" name="FWCoreReflectionTestObjects">
+  <flags LCG_DICT_HEADER="stubs/classes.h"/>
+  <flags LCG_DICT_XML="stubs/classes_def.xml"/>
+  <use name="DataFormats/Common"/>
+</library>
+
+<test name="TestFWCoreReflectionCheckClassVersion" command="run_checkClassVersion.sh"/>
```

Comment:
@smuzaffar By the way, is there a away to denote a dependence from a test to the build rule that copies `scripts/*` to `$CMSSW_VERSION/bin/$SCRAM_ARCH`? Would e.g.
```xml
<test name="TestFWCoreReflectionCheckClassVersion" command="run_checkClassVersion.sh">
  <use name="FWCore/Reflection"/>
</test>
```
do that?

Reply:
No @makortel , currently there is no way to provide such dependency

Reply:
@makortel , it should be trivial to add such a support e.g. running the unit test rule can depend on `bin` and `script` rules of its dependencies

Reply:
Ok, thanks. I don't think the support would be crucial for this PR, but would be nice to have at some point.